htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -541,6 +541,7 @@ function fetch($id, $ref='',$ref_ext='')
                 // Properties of parent table llx_c_actioncomm (will be deprecated in future)
                 $this->type_id   = $obj->type_id;
                 $this->type_code = $obj->type_code;
+                $this->color = $obj->color;
```

Comment:
Don't you mean   $this->type_color = $obj->type_color ?  (later you use $this->type_color)